includes/class-wc-payments-order-service.php

```diff
@@ -1272,7 +1295,7 @@ private function generate_dispute_closed_note( $dispute_id, $status ) {
 	 *
 	 * @return string Dispute details page url.
 	 */
-	private function compose_dispute_url( $dispute_id ) {
+	public function compose_dispute_url( $dispute_id ) {
```

Comment:
This can be changed back to `private` since it is no longer being used in this PR.
```suggestion
	private function compose_dispute_url( $dispute_id ) {
```